### DISCUSSION TITLE
What to put in Issues

### DISCUSSION THREAD

**Discussion by ceford** [max123kl/J4.docs#5] [General]
When an issue is opened I think it is important to look at the PR to see which documentation needs creating or updating. It could be Help page, a User Tutorial page or a Developer Information page. In this case the PR says:

If an existing document can be identified there should be a link to it. Otherwise ... what?

**Comment by max123kl** [comment]
My preference would be to set up a bot in the CMS repo that generates an issue in this repo whenever documentation is requested in a PR. It would be ideal if there were tags or labels at the same time that reference either the user or dev manual or help page.

**Reply by alann60** [reply]
This seems a good approach to bring in issues automatically. If there is a way to set it's status automatically to "To Do" it would start the workflow process.

**Comment by carlitorweb** [comment]
The Kanban project will be enough for us to manage our internal workflow (New documentation needed by a PR. What documentation to upgrade. What to change. What to translate etc...) we can have all there.

In my opinion we should let Issues section, for others discussion like someone outside the team come with a idea about a tutorial or asking help about how edit something etc.

Pull request tab in my opinion we should let for people can "also" send a markdown file with a new brand tutorial or a new documentation (in case they not want enter to mediawiki, who is our current platform).
Will be double work for us since we need copy/paste the tutorial + do the formatting for mediawiki. (I hope in a future we can migrate all for a markdown formatting, will be all a lot more easy) This is a way we can take for not lose the contribution, who is what we need at the end

**Reply by alann60** [reply]
Having had a quick play with the Project / Kanban board, you can change the status in the Issue that positions the task on the board.

If the new issue is picked up in the repo the necessary labels could be added there. The status could then be moved on to another ready to be picked up by a writer who will then move it along, updating statuses as they go.

I'm mindful that not everyone works with Markdown so it should not be the only format. Even a plain text file would probably be sufficient?

**Reply by carlitorweb** [reply]
Yes, plain text also can work. But we need add the markdown format after then

**Reply by max123kl** [reply]
Our main goal, after all, is to make it easier for as many users as possible to access the docs and thus attract contributors.

If I understand Cliff correctly, it costs no (or not much) effort to convert incoming text to Markdown (or any other markup). Therefore, I think that any format that can be converted without any problem should be accepted.
